packages/wdio-sauce-service/src/api.ts

```diff
@@ -0,0 +1,52 @@
+import * as util from 'node:util'
+import logger from '@wdio/logger'
+import type { AxiosInstance } from 'axios'
+import axios, { isAxiosError } from 'axios'
+
+import type { Region, TestRunRequestBody, HTTPValidationError } from './types.js'
+import { apiURLMap } from './types.js'
+
+const log = logger('@wdio/sauce-service')
+
+export class TestRuns {
+    private api: AxiosInstance
+
+    constructor(opts: { username: string; accessKey: string; region: Region }) {
+        this.api = axios.create({
+            auth: {
+                username: opts.username,
+                password: opts.accessKey,
+            },
+            baseURL: apiURLMap.get(opts.region),
+        })
+    }
+
+    async create(testRuns: TestRunRequestBody[]) {
+        try {
+            log.debug('Submitting test run to test-runs api', testRuns)
+            await this.api.post<void>('/test-runs/v1/', {
```

Comment:
This should be part of the `saucelabs` NPM package which already has a lot of primitives to expose API requests in a simple interface.

Reply:
Good call! I'll move it out.